src/main/scala/PrivateHome/data.scala

```diff
@@ -1,47 +1,180 @@
 package PrivateHome
 
+import PrivateHome.Devices.MHz.mhzSwitch
+import PrivateHome.Devices.MQTT.mqttSwitch
 import PrivateHome.Devices.Switch
+import scalikejdbc._
 
-import scala.xml._
 
 object data {
+  /**
+   * An map containing all settings
+   */
+  val settings: Map[String, Int] = Map(("port", 2888), ("ip", 565))
+
+
+  GlobalSettings.loggingSQLAndTime = LoggingSQLAndTimeSettings(
+    enabled = false,
+    singleLineMode = false,
+    printUnprocessedStackTrace = false,
+    stackTraceDepth = 15,
+    logLevel = Symbol("debug"),
+    warningEnabled = true,
+    warningThresholdMillis = 3000L,
+    warningLogLevel = Symbol("warn")
+  )
+
+
+  Class.forName("org.h2.Driver")
+  ConnectionPool.singleton("jdbc:h2:./daten/Devices", "user", "pass")
```

Comment:
Please add config option for username, password and database name. It is necessary for getting security.

Reply:
I will not add an configuration for the database name because this isn't necesary